openedx/core/djangoapps/content/course_overviews/tests/__init__.py

```diff
@@ -35,7 +35,7 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory, check_mongo_calls, check_mongo_calls_range
 
-from .models import CourseOverview, CourseOverviewImageSet, CourseOverviewImageConfig
+from ..models import CourseOverview, CourseOverviewImageSet, CourseOverviewImageConfig
```

Comment:
I wanted `tests` to be a module, because I added `factories.py` to it. But I could move the tests into a separate submodule as well.